vita3k/gui/src/settings_dialog.cpp

```diff
@@ -103,9 +103,10 @@ static bool clear_and_refresh_game_list(HostState &host, GuiState &gui) {
 
 using namespace list;
 void draw_settings_dialog(GuiState &gui, HostState &host) {
+    const ImGuiWindowFlags settings_flags = ImGuiWindowFlags_AlwaysAutoResize;
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
-    ImGui::Begin("Settings", &gui.configuration_menu.settings_dialog, ImGuiWindowFlags_AlwaysAutoResize);
-    const auto settings_tab_flags = ImGuiTabBarFlags_None;
+    ImGui::Begin("Settings", &gui.configuration_menu.settings_dialog, settings_flags);
```

Comment:
I'm not totally sure why you moved this out. If you say it's for a cleaner look, cool!

Reply:
based on admin works on demo imgui ^^